packages/core/src/state/StateSystem.ts

```diff
@@ -12,89 +12,79 @@ const DEPTH_TEST = 3;
 const WINDING = 4;
 const DEPTH_MASK = 5;
 
-/**
- * System plugin to the renderer to manage WebGL state machines.
- *
- * @class
- * @extends PIXI.System
- * @memberof PIXI
- */
+/** System plugin to the renderer to manage WebGL state machines. */
```

Comment:
This requires `@memberof PIXI` or else it will not show up in the correct namespace.

https://pixijs.download/chore/docs-StateSystem/docs/StateSystem.html